### ISSUE TITLE
Caliper documentation about grafana needs correcting

### ISSUE THREAD

**Issue by davidkel** [hyperledger/caliper#1437]
The current documentation has `grafana visualization` under the transaction section for resource and transaction monitors but it shouldn't be there.

The current text states

```
Grafana is an analytics platform that may be used to query and visualize metrics collected by Prometheus. Caliper clients make the following metrics available, either via a direct scrape or indirectly via a Prometheus Push Gateway:

caliper_tx_submitted (counter)
caliper_tx_finished (counter)
caliper_tx_e2e_latency (histogram)
Each of the above are tagged with the following default labels:

roundLabel: the current test round label
roundIndex: the current test round index
workerIndex: the zero based worker index that is sending the information
We are currently working on a Grafana dashboard to give you immediate access to the metrics published above, but in the interim please feel free to create custom queries to view the above metrics that are accessible in real time.
```

it should be removed and perhaps taking the information about labels that are default and putting it into the prometheus transaction monitor information. Also there is no work being done to create a Grafana dashboard.

**Comment by anushatomar13** [comment]
Hi @davidkel ! Can I work on this issue?

**Comment by davidkel** [comment]
Now fixed in 0.6.0 and vNext. Thank you for the contribution
